.github/workflows/release.yaml

```diff
@@ -267,31 +261,66 @@ jobs:
         run: |
           hack/build/ci/install-cgo-dependencies.sh
       - name: Generate release notes
-        id: generate_notes
+        shell: bash
+        env:
+          PRE_RELEASE: ${{ contains(github.ref, '-rc.') }}
         run: |
           hack/build/ci/generate-release-notes.sh
       - name: Generate manifests
         env:
-          IMAGE: ${{ matrix.url }}/${{ secrets[matrix.repository] }}
-          TAG: ${{ needs.prepare.outputs.version }}
-          CHART_VERSION: ${{ needs.prepare.outputs.version }}
-        run: make manifests
-      - name: Rename manifests
+          version: ${{ needs.prepare.outputs.version }}
+          platforms: |
+            kubernetes
+            openshift
+          images: |
+            docker.io/dynatrace/dynatrace-operator
```

Comment:
These should be taken from the Secrets

Reply:
I wrote it here directly, as I do not see the need to put it in a secret, as this information is publicly available.

Where do you see the advantage in using a secret?